python.md

```diff
@@ -83,6 +84,18 @@ with the convention unless you have a very good reason not to.
 This convention also applies to variable names.
 
 
+### Model method naming conventions
+
+When sorting is involved, here are some guidelines to keep things consistent:
+
+- Prefer "active" to "current", ie `get_active_direct_debit_instruction`
```

Comment:
I wonder if this one might be horses for courses, as 'active' can have other meanings in energy. For example, the 'current' meter point attached to an account might not be 'active', so you might want to stick with 'get_current_meter_point' rather than 'get_active_meter_point'. 

Is there a more general reason to prefer 'active' to 'current', or are we just wanting to pick one and standardise?

Reply:
> I wonder if this one might be horses for courses, as 'active' can have other meanings in energy. For example, the 'current' meter point attached to an account might not be 'active', so you might want to stick with 'get_current_meter_point' rather than 'get_active_meter_point'.

In that instance, what does 'current' actually mean?

> Is there a more general reason to prefer 'active' to 'current', or are we just wanting to pick one and standardise?

Just the latter.

Reply:
> In that instance, what does 'current' actually mean?

It would be the meter point associated to the account at that point in time. A better example might be the accounts -> properties relationship. If I say `property.get_active_account()` and get back an account whose status is `DORMANT` rather than `ACTIVE`, I might be surprised, whereas `property.get_current_account()` doesn't give me any expectations about the account I'm getting back, except that it's currently associated with the property.

The wider point here I would say is that 'current' is pretty unambiguous in meaning 'now', whereas 'active' suggests activity in its subject, which will not always be true, as in the dormant but current account example above.